website/pages/blog/product-updates-7.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "CloudQuery Product Updates #7"
+tag: product
+date: 2023/08/21
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here!
+
+TLDR:
+
+- Open source: SDKs & Plugins.
+- Premium (closed source): New Plugins, Policies and dashboards.
+
+## SDKs!
+
+With recent Adoption of [Apache Arrow](https://www.cloudquery.io/blog/adopting-apache-arrow-at-cloudquery) as CloudQuery
+in memory data form CloudQuery SDK is now expanding beyond Go (which means you can write CloudQuery plugins in other languages)! The first one is of-course the king of data engineering - Python!
+
+Check out our [Python announcement blog](https://www.cloudquery.io/blog/announcing-cloudquery-python-sdk)!
```

Comment:
We could link to https://www.cloudquery.io/docs/developers/creating-new-plugin/python-source and https://www.cloudquery.io/docs/developers/creating-new-plugin/javascript-source too

Reply:
Nice! I wasn't aware javascript already had a guide - we should also schedule some socials around that.